### DISCUSSION TITLE
No Admin-account, ADMIN_TOKEN doesn't work

### DISCUSSION THREAD

**Discussion by Detlef-AC** [dani-garcia/vaultwarden#2478] [Q&A]
I installed bitwarden_rs, but I can't login.

I tried these variables in different configurations:
      - SIGNUPS_ALLOWED=true
      - DISABLE_ADMIN_TOKEN=false
      - INVITATIONS_ALLOWED=true
      - ADMIN_TOKEN=<secret token>

It's not possible, to get the admin-portal via https://<myDomain>/admin
I only get the login-dialog. Because no user has been created yet, I can't log in, of course.

Any idea? Thanks.

**Comment by BlackDex** [comment]
The admin portal has no connection to any users created for the vault it self.
To access the admin portal you need to provide an `ADMIN_TOKEN=SomeRandomString` or set `DISABLE_ADMIN_TOKEN=true`.
After setting those you should be able to access the admin portal by going to your domain, for example `https://vaultwarden.domain.tld/admin` Make sure you do not have `/#/admin` since that is the web-vault interface it self and not the admin portal.

**Reply by Detlef-AC** [reply]
In my docker-compose.yml I kept as the only entry on the environment ADMIN_TOKEN=<SomeRandomString>. I stopped docker-compose, deleted the data-Dir and restarted docker-compose.
And again, with https://<mydomain.de>/admin I got the login - dialog, not the admin portal and not the request for the ADMIN_TOKEN; the bitwarden-desktopclient refused "create account" with "401 unauthorized", which is also correct.
I have no idea how to get on.

**Reply by Detlef-AC** [reply]
Sorry: I used a random string after ADMIN_TOKEN=

**Reply by BlackDex** [reply]
Then i suggest that you double check your reverse proxy configuration.
https://github.com/dani-garcia/vaultwarden/wiki/Proxy-examples

**Reply by sysadminpower2019** [reply]
you need to generate the file yourself.  https://docs.nginx.com/nginx/admin-guide/security-controls/configuring-http-basic-authentication/

**Reply by Detlef-AC** [reply]
SOLVED

I had to commend out this line:
#- "traefik.http.routers.bitwarden-secure.middlewares=traefik-auth"

Thanks to all.
